File: date 12.02.2026/palindrome.java
public class palindrome{
    static int reversenumber(int n){
        int rev=0;
        while(n>0){
            rev=rev*10+n%10;
            n=n/10;

        }
        return rev;
    }
        public static void main(String[] args){
            int n_1=121;
            int revs=reversenumber(n_1);
            System.out.println("reverse of n="+revs);
            if (n_1==revs){
                System.out.println("palindrome number");
            }
            else{
                System.out.println("not a palindrome number");
            }
        }

    }
    

File: date 14-03-2026/palindrome.py
def number_palindrome(n):
    original = n
    reverse = 0

    while n > 0:
        digit = n % 10
        reverse = reverse * 10 + digit
        n = n // 10

    if original == reverse:
        print("Palindrome Number")
    else:
        print("Not a Palindrome Number")

num = int(input("Enter a number: "))
number_palindrome(num)